README and CONTRIBUTION

Vegter opened the pull request:
Fixes #1

Vegter committed:
Initial Dutch and English README
Add .idea (IntelliJ) to gitignore

Vegter committed:
opbouw and structure instead of werkwijze and method
link to Englisch README

Vegter committed:
Initial contributing document

Vegter committed:
Add reference to Dutch README in English README

Vegter committed:
Add reference to GOB Documentation for technical documentation and getting started

MauriceHendriks commented on README.en.md:
```diff
@@ -0,0 +1,89 @@
+# GOB
+
+Generic Disclosure of Basic Data
```
Generic Disclosure of `Key Registrations`

MauriceHendriks commented on README.en.md:
```diff
@@ -0,0 +1,89 @@
+
+## Target
+
+The aim of the project is to guarantee the delivery of high-quality basic data for municipal processes against minimal management efforts and costs.
```
Misschien beter om hier ook `target` gebruiken.

MauriceHendriks commented on README.en.md:
```diff
@@ -0,0 +1,89 @@
+## Background
+
+Basic data is the raw material for a large number of municipal processes.
+Without these data, the processes can not be executed.
```
Hierboven schrijf je data in enkelvoud. Daarna in meervoud. Ik zou ook zeggen `these processes` om te verwijzen naar je vorige zin.

MauriceHendriks commented on README.en.md:
```diff
@@ -0,0 +1,89 @@
+
+Not all systems can handle the different deliveries.
+A 'translation' is therefore necessary.
+Within GOB, the basic data from the various source registrations are stored, processed and distributed in a format as defined in
```
Gaat het hier over de vorm waarin werd gecommuniceerd zoals `StUF` of `UVA` of over de de inhoud van de berichten? Stelselpedia gaat namelijk maar beperkt in op de vorm, wel op de inhoud.

Vegter replied:
Weliswaar een logisch gegevensmodel maar daar is wel een technisch datamodel van af te leiden dat is opgesteld volgens de naamgevingen in Stelselpedia en RSGB

MauriceHendriks replied:
Probleem is alleen dat wanneer het logisch model leidend is voor wat de GOB implementeert je mogelijk tegen dezelfde problemen aanloopt die er nu ook met bijvoorbeeld de StUF implementatie zijn.

Vegter replied:
Mee eens, het model is daarom ook als indicatief te zien, het is zeker niet op te vatten als beperkt tot. Maar je kunt ervan uitgaan dat wat in Stelselpedia staat ook onder een min of meer vergelijkbare naam te vinden zal zijn in GOB. En daarnaast nog wel meer data. Denk bijvoorbeeld aan CBS namen voor gebieden.
Daarnaast passen we versioning toe zodat we niet opgesloten worden in welk formaat dan ook maar vrijheid hebben om door te groeien.

MauriceHendriks replied:
De kern is voornamelijk dat de oude StUF is gemaakt voor het synchroon houden van systemen waarbij niet wordt van wordt uitgegaan dat de ontvanger de historische continuiteit probeert te waarborgen. Dat kan met de StUF (impelementatie van Amstedam namelijk maar beperkt). Dat heeft dus wezenlijke invloed op hoe er `by-design` wordt nagedacht over de implementatie (en invulling) van berichten. Hetzelfde speelt er ook bij de BAG wat betreft de UVA2 leveringen. Die worden dan wel historisch aangeboden, maar missen informatie om een feilloze historisch consistente schaduw op te zetten. De vrees is dat wanneer deze aspecten dus niet vanaf de ontwerp tafel meegenomen worden we in een implementatie terecht komen met dezelfde beperkingen als nu.

Vegter replied:
Goed dat je dit opmerkt. Historie is nu juist een integraal onderdeel van GOB. Feitelijk is de architectuur gebaseerd op een boekhouding, met journaalposten en rapportages. Door het vastleggen van alle mutaties als journaalposten is het model slechts een afgeleide geworden. Historisch snapshots, tussentijdse rapportages, het is allemaal mogelijk. En ook een undo, dat is namelijk een normale gang van zaken in een boekhouding, gewoon correctieposten maken. Mocht je toegang hebben tot docuwiki nodig ik je uit om kennis te nemen van de details van de architectuur en alle overwegingen die aan de architectuur ten grondslag liggen.

spreeker replied:
Wow wat een requirements. Ik ben wel nieuwsgierig geworden. Een correcte feilloze huidige presentatie was/is al moeilijk genoeg!

MauriceHendriks replied:
> dat is namelijk een normale gang van zaken in een boekhouding, gewoon correctieposten maken.

En het is goed dat die aanname een requirement is geworden! Ik hoop dat dat ook getoetst wordt door de partijen die er op die manier van afhankelijk zijn. Zoals onze eigen afdeling Basisstatistiek. Voorheen was die aanname er namelijk ook (zoals je zelf al zegt), maar is dat nooit tot een sluitende implementatie gekomen.

> Mocht je toegang hebben tot docuwiki nodig ik je uit om kennis te nemen van de details van de architectuur en alle overwegingen die aan de architectuur ten grondslag liggen.

Ga ik zeker doen. Dank.

Vegter committed:
Rewrite basic data to basic data that is held by key registrations

Vegter committed:
Let the english documentation be the leading README documentation

bvhme commented on .gitignore:
```diff
@@ -0,0 +1 @@
+/.idea
```
Shouldn't this be in an individuals global gitignore?

Vegter replied:
Right, deleted gitignore for now

bvhme commented on README.md:
```diff
@@ -1,2 +1,89 @@
+The seven key registers are:
+1. persons (BRP)
+2. addresses and buildings (BAG)
+3. value of immovable property (BR WOZ)
```
Ontroerend Goed is 'real estate' in English.

Vegter replied:
Done

bvhme commented on README.md:
```diff
@@ -1,2 +1,89 @@
+
+Three systems are in use for retrieving data from the source or the national facility, the management, the translation and the distribution of basic data:
+- DIVA for real estate data
+- Makelaar suite for personal details
```
Isn't this `Makelaarssuite`?

Vegter replied:
Right, changed it

bvhme commented on README.md:
```diff
@@ -1,2 +1,89 @@
+
+Not all systems can handle the different deliveries.
+A 'translation' is therefore necessary.
+Within GOB, basic data from the various source registrations are stored, processed and distributed in a format as defined in
```
add: `the standards set by the City of Amsterdam and the association of Dutch municipalities`

Vegter replied:
Done

bvhme commented on README.nl.md:
```diff
@@ -0,0 +1,91 @@
+
+**G**enerieke **O**ntsluiting **B**asisgegevens
+
+Let op:
```
Great! I always like to add a ⚠️ here, with `:warning:`

bvhme reviewed (approved):
Looks great! Some minor changes suggested.

Vegter committed:
Text corrections

The English readme contained some incorrect texts. Makelaarsuite was
written as Makelaar suite, The standards of the Dutch City of
Amsterdam was not mentioned.

Vegter committed:
Text corrections

Include the standards set by the City of Amsterdam and the association
of Dutch municipalities.

Immovable property => real estate

Vegter commented:
All comments have been processed. @bvhme any more comments?

bvhme reviewed (approved):
I think it is great!

bvhme commented:
@Vegter Shall I merge these kind of PRs or stay out of it?

Vegter commented:
Hi @bvhme Please merge if you are OK with it.
